chaosblade-exec-plugin/chaosblade-exec-plugin-jvm/src/main/java/com/alibaba/chaosblade/exec/plugin/jvm/JvmModelSpec.java

```diff
@@ -92,7 +94,7 @@ public void preCreate(String uid, Model model) throws ExperimentException {
             try {
                 ((DirectlyInjectionAction) actionSpec).createInjection(uid, model);
             } catch (Exception e) {
-                throw new ExperimentException("create injection failed:" + e.getMessage());
+                throw new ExperimentException("create injection failed: " + Arrays.toString(e.getStackTrace()) + ", " + e, e);
```

Comment:
e.toString() enough

Reply:
But it doesn't work, no more valid information could be get if using e.toString().

Reply:
It is not recommended to do this, usually we only need to return  e.getMessage(), but we can print error e here